2996_reflection/reflection.md

```diff
@@ -81,21 +110,73 @@ namespace std {
 
 In our initial proposal a value of type `std::meta::info` can represent:
 
-  - an error
+  - an error (corresponding to an "invalid reflection")
   - any (C++) type and type-alias
   - any function or member function
   - any namespace-scope variable or any C++ static data member
   - any non-static data member
   - any constant value
   - any template
+  - any namespace
 
 [ (DV) I'm currently excluding local variable representation.  We can change that if we have a use for them that we feel must be in the initial core functionality. ]
 
+
+The type `std::meta::info` is a _scalar_ type with associated namespace `std::meta`.  Nontype template arguments of type `std::meta::info` are permitted.
```

Comment:
Just for clarity, let's add a parenthetical (or separate sentence, whatever) that means that you can make unqualified calls to the library functions in `std::meta`. Maybe even add that, like, I dunno what's simple...  `std::meta::add_pointer(^int)` and `add_pointer(^int)` are both valid and mean the same thing.